content/en/continuous_integration/pipelines/awscodepipeline.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Set up Tracing on a AWS CodePipeline Pipeline
+kind: documentation
+aliases:
+  - /continuous_integration/setup_pipelines/codepipeline
+further_reading:
+    - link: "/continuous_integration/pipelines"
+      tag: "Documentation"
+      text: "Explore Pipeline Execution Results and Performance"
+    - link: "/continuous_integration/troubleshooting/"
+      tag: "Documentation"
+      text: "Troubleshooting CI"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+## Compatibility
+
+- **Partial pipelines**: View partially retried executions
+
+## Configure the Datadog integration
+
+To set up the integration between [AWS CodePipeline][1] and Datadog CI Visibility, two AWS resources need to be created:
```

Comment:
```suggestion
To set up the integration between [AWS CodePipeline][1] and Datadog CI Visibility, create two AWS resources  during the EventBridge Rule creation process:
```

Reply:
Mmm I would prefer to not have `during the EventBridge Rule creation process` there, as also in the guide they are created separately. So we could go for an hybrid like:

```suggestion
To set up the integration between [AWS CodePipeline][1] and Datadog CI Visibility, create two AWS resources:
```

?